xbmc/utils/DatabaseUtils.cpp

```diff
@@ -299,6 +299,7 @@ std::string DatabaseUtils::GetField(Field field, const MediaType &mediaType, Dat
       result = StringUtils::Format("episode_view.c{:02}", VIDEODB_ID_EPISODE_EPISODE);
     else if (field == FieldUniqueId)
       result = StringUtils::Format("episode_view.c{:02}", VIDEODB_ID_EPISODE_IDENT_ID);
+    else if (field == FieldUniqueIdValue) return "episode_view.uniqueid_value";
```

Comment:
Isn't something like this also necessary for the other tables like `movies` etc?

Reply:
Good catch. Added.